content/post/mnist-example.md

```diff
@@ -0,0 +1,158 @@
++++
+title = "A Scikit-learn pipeline in Wallaroo"
+draft = false
+date = 2017-11-16T00:00:00Z
+tags = [
+    "wallaroo",
+    "machine learning",
+    "python",
+    "example",
+    "tutorial"
+]
+categories = [
+    "Machine Learning"
+]
+description = "Creating an inference pipeline with MNIST"
+author = "amosca"
++++
+
+Whilst it would seem that machine learning is taking over the world, a lot of the attention has been focused towards researching new methods and applications, and how to make a single model faster. At Wallaroo Labs we believe that, in order to make the benefits of machine learning truly ubiquitous, there needs to be a significant improvement in how we put those impressive models into production. This is where the stream computing paradigm becomes useful: as for any other type of computation, we can use streaming to apply machine learning models to a large quantity of incoming data, using available techniques in distributed computing.
+
+Nowadays, many applications with streaming data are either applying machine learning or have a very good use case for it. In this example, we will explore how we can build a machine learning inside Wallaroo, our high performance stream processing engine, to classify images from the [MNIST dataset](ihttp://yann.lecun.com/exdb/mnist/), using a basic two-stage model in Python. Whilst recognizing hand-written digits is a practically solved problem, even a simple example like the one we are presenting provides a real use case (imagine automated cheque reading in a large bank), and the same setup can be used as a starting point for virtually any machine learning application - just replace the model.
```

Comment:
The link to the MINST dataset (`ihttp://yann.lecun.com/exdb/mnist/`) seems to be incorrect. I'm guessing it should be `http://yann.lecun.com/exdb/mnist/`